### DISCUSSION TITLE
UI 设计想法

### DISCUSSION THREAD

**Discussion by Daydreamer-riri** [pot-app/pot-desktop#901] [Ideas]
最好的方案我认为是类似 Devtoys 2.0 的方案，根据不同的平台相同的组件应用不同的样式，使应用更具有本机风格。Windows 可以使用云母材质。

这样也不需要专业的 UI 成员，各个平台都有自己的规范，以及现成的样式。

![image](https://github.com/user-attachments/assets/d440397b-0232-40b1-b70b-e388fa3d5cdf)
![image](https://github.com/user-attachments/assets/7601a273-9263-4f46-8b69-3964dfaff0cf)
![image](https://github.com/user-attachments/assets/08239c6c-7eac-4ae4-b3f2-35dc2a620a64)

相关阅读链接：https://devtoys.app/blog/the-journey-to-devtoys-2.0

**Comment by elarbor** [comment]
嗐，就是个辅助小工具。现在这样其实也挺好的
